E2-Breakdown-Money/src/main/java/com/redelles/e2/Utils.java

```diff
@@ -4,20 +4,71 @@
 
 public class Utils {
```

Comment:
All of this file contains methods to get numbers from Scanner. Utils is a too generic name which finally it will be a pool of shit, with thousand methods. Change the name to have one more properly